pkgs/development/python-modules/jaxlib/bin.nix

```diff
@@ -231,15 +238,18 @@ buildPythonPackage {
     license = licenses.asl20;
     maintainers = with maintainers; [ samuela ];
     platforms = [
+      "aarch64-linux"
       "aarch64-darwin"
       "x86_64-linux"
       "x86_64-darwin"
     ];
```

Comment:
```suggestion
```
Shouldn't we now remove the `platforms` attribute entirely as they are now all supported (which is the default) ?